app/v2/services/AuditingService.scala

```diff
@@ -43,4 +45,17 @@ class AuditingServiceImpl @Inject() (auditingConnector: AuditingConnector) exten
       case NonFatal(e) =>
         logger.warn("Unable to audit payload due to an exception", e)
     }
+
+  def getAuditType(messageType: String): EitherT[Future, FailedToValidateError, AuditType] =
+    EitherT {
+
+      AuditType.values
+        .filterNot(
+          auditType => messageType.equals("DeclarationData")
+        )
+        .find(_.name == messageType) match {
+        case None            => Future.successful(Left(FailedToValidateError.InvalidMessageTypeError(s"Invalid Message Type for auditing : $messageType")))
```

Comment:
This again is not a validation error, so we should not be using `FailedToValidateError`s here.

Reply:
removed complete method from the Audit service not required